code/obj/item/bell.dm

```diff
@@ -17,6 +18,9 @@
 		return
 	src.visible_message("<span class='notice'><b>[user]</b> rings \the [src]!</span>")
 	playsound(src, 'sound/effects/bell_ring.ogg', 30, 0)
+	if(sends_signal_to_hop_watch)
+		for(var/obj/item/pocketwatch/watch in world)
```

Comment:
iterating world is Bad, use `for_by_tcl` instead (you'll need to add `START_TRACKING` to the watch's `New` and `STOP_TRACKING` to its `disposing`)